template.tpl

```diff
@@ -56,23 +324,45 @@ const setInWindow = require('setInWindow');
 const createArgumentsQueue = require('createArgumentsQueue');
 const copyFromWindow = require('copyFromWindow');
 const callInWindow = require('callInWindow');
+const log = require('logToConsole');
 
+log("data", data);
```

Comment:
@rachelpeles i think we can remove that don't you think?

Reply:
yeah, I removed it 
thanks!